### ISSUE TITLE
Not keyboard-only accessible and missing ARIA attributes

### ISSUE THREAD

**Issue by designbymichael** [marioloncarek/megamenu-js#20]
All your menu elements are missing proper ARIA attributes for accessibility, and the mega menu appears to require mouseover functionality to expand.  For accessibility reasons, you should be able to navigate through the menu exclusively with a keyboard.
